### ISSUE TITLE
Try out Git helper functions in FSI

### ISSUE THREAD

**Issue by halcwb** [fsharp/FAKE#2472]
### Description

I want to try out some Git helper functions in the FSI.

### Repro steps

1. Create a load script using `paket generate-load-scripts` to load the FAKE dlls.

2. Set the current directory: `System.Environment.CurrentDirectory <- __SOURCE_DIRECTORY__`

3. Try out a Git helper function like: `Git.Information.getBranchName "."`

### Expected behavior

The current branch name is retrieved.

### Actual behavior

The git command cannot be run:

> System.Exception: Start of process 'git.exe' failed. ---> System.ComponentModel.Win32Exception: The system cannot find the file

### Known workarounds

None

### Related information 

FAKE 5 - F# Make (5.19.0) 
Paket.Core: 5.241.2
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100
 Commit:    04339c3a26

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100\

Host (useful for support):
  Version: 3.0.0
  Commit:  7d57652f33

.NET Core SDKs installed:
  2.1.801 [C:\Program Files\dotnet\sdk]
  2.1.802 [C:\Program Files\dotnet\sdk]
  2.2.401 [C:\Program Files\dotnet\sdk]
  2.2.402 [C:\Program Files\dotnet\sdk]
  3.0.100 [C:\Program Files\dotnet\sdk]

**Comment by matthid** [comment]
Can you give a full example of what you are trying to do? I guess you either need to have git in PATH or something else is missing.

**Comment by halcwb** [comment]
What I want to accomplish is to try out helper functions for a build script in the FSI. I read and used the following in your manual: 

## Run script without fake.exe (via fsi)

FAKE 5 scripts are not hardwired to the new FAKE 5 runner (even if they look like with the new header). To make them work with FSI all you need to do is to setup a "FAKE-Context":

```fsharp
// Regular header and `#load ".fake/build.fsx/intellisense.fsx"`

#if !FAKE
let execContext = Fake.Core.Context.FakeExecutionContext.Create false "build.fsx" []
Fake.Core.Context.setExecutionContext (Fake.Core.Context.RuntimeContext.Fake execContext)
#endif

// Your Script code...
```
So, I also tried that approach with no luck. 

Basically, I want to create a build script like you would write any F# code. Write some code and then select it and send it to the FSI.

**Comment by matthid** [comment]
Yes, what you are trying to do should work.
The fake-context is not always required (depends a bit on the module code)
Can you provide an example? I guess the problem is just that the module cannot find `git.exe` (this is what I meant in my initial response with `PATH`)

**Comment by halcwb** [comment]
Indeed I think git is not found (although globally installed on my machine). The code snippet that fails is:

```
let isReleaseBranchCheck () =
    if Git.Information.getBranchName "" <> releaseBranch then failwithf "Not on %s.  If you want to release please switch to this branch." releaseBranch

```

I get the error message. If I run the same code using the fake runner, it all works.

**Comment by halcwb** [comment]
Sorry, my bad, didn't have git in the PATH.
